package.json

```diff
@@ -78,5 +78,8 @@
     "singleQuote": false,
     "trailingComma": "es5",
     "bracketSpacing": true
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
A scope package is always assumed to be private unless the `access` property is set to  `public`.